packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -58,6 +61,35 @@ class Send extends Component {
     }
   });
 
+  calculateMax = (gas, gasPrice) => {
+    const { token, balance } = this.props;
+    const gasBn = gas ? new BigNumber(gas) : new BigNumber(21000);
+    const gasPriceBn = new BigNumber(gasPrice);
+    let output = new BigNumber(gasPrice);
+
+    if (token.address === 'ETH') {
+      output = fromWei(
+        toWei(balance).minus(gasBn.mul(toWei(gasPriceBn, 'shannon')))
```

Comment:
I would prefer to merge #274 first, because the syntax is changed to `.multipliedBy`

Reply:
Sure go ahead